packages/app-accounts/src/Editor.tsx

```diff
@@ -98,21 +100,36 @@ class Editor extends React.PureComponent<Props, State> {
     const { currentPair, defaultValue, editedName } = this.state;
 
     if (!currentPair) {
-      return t('editor.none', {
-        defaultValue: 'There are no saved accounts. Add some first.'
-      });
+      return (
+        <div>
+          <div>{t('There are no saved accounts. Create an account or upload a JSON file of a saved account.')}</div>
+          <div className={'accounts--Address-wrapper'}>
+            <div className={'accounts--Address-file'}>
+              <UploadButton size={'big'} onChangeAccount={this.onChangeAccount} />
+            </div>
+          </div>
+        </div>
+      );
     }
 
     const address = currentPair.address();
 
     return (
-      <div className='ui--grid'>
-        <AddressSummary
-          className='shrink'
-          value={address}
-        />
-        <div className='grow'>
-          <div className='ui--row'>
+      <div className={'accounts--flex-group-row'}>
```

Comment:
No. `className='accounts--flex-grow'`

Reply:
Sorry I don't quite understand. Are you suggesting I just rename that className to `accounts--flex-grow`, or would you like me to somehow incorporate `flex-grow` https://developer.mozilla.org/en-US/docs/Web/CSS/flex-grow?

Reply:
Now I get the comment.

`<div className='accounts--flex-group-row'>` (without the wrapping {}'s)